crates/codegen/runtime/cargo/src/runtime/parse_error.rs

```diff
@@ -1,7 +1,9 @@
 use std::collections::BTreeSet;
 
+use metaslang_cst::AbstractKind as _;
```

Comment:
I suggest not using the `as _` syntax, as it prevents dead-code checking if it is no longer needed.

```suggestion
use metaslang_cst::AbstractKind;
```

Reply:
Removed in the rebase. However I prefer this when importing a trait that isn't directly referenced.